notebooks/community/sdk/sdk_automl_forecasting_evaluating_a_model.ipynb

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        column_transformations=[
</pre><p>Use column_specs instead of column_transformations (deprecated)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Done.